third_party/terraform/website/docs/r/google_project_iam_custom_role.html.markdown

```diff
@@ -49,7 +49,7 @@ The following arguments are supported:
 
 * `stage` - (Optional) The current launch stage of the role.
     Defaults to `GA`.
-    List of possible stages is [here](https://cloud.google.com/iam/reference/rest/v1/organizations.roles#Role.RoleLaunchStage).
+    List of possible stages is [here](https://cloud.google.com/iam/reference/rest/v1/projects.roles#Role.RoleLaunchStage).
```

Comment:
Looks like only the "organizations" version of the link works, this version doesn't point to role launch stages